tailwind/plugin.js

```
@@ -390,10 +390,226 @@ export default plugin(
+                  fontWeight: 'inherit',
+                  fontSize: 'inherit',
+                },
+
+                // blockquote: left border, receded color, no italic, no quote marks
+                blockquote: {
+                  'border-inline-start-width': '2px',
+                  borderInlineStartColor: 'var(--ink-gray-3)',
+                  borderInlineStartStyle: 'solid',
+                  fontStyle: 'normal',
+                  color: 'var(--ink-gray-6)',
+                },
```

Biome (2.4.10)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)